hudi-common/src/main/java/org/apache/hudi/common/storage/StorageSchemes.java

```diff
@@ -36,6 +36,10 @@
   S3A("s3a", false), S3("s3", false),
   // Google Cloud Storage
   GCS("gs", false),
+  // Azure WASB
+  WASB("wasb", false), WASBS("wasbs", false),
```

Comment:
https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlFileSystem.java#L445
https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azure/NativeAzureFileSystem.java#L1575
https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java#L246

All suggest append support? Stumbled onto it as I was trying to read up on this.. 

actually, this is so awesome, if it has append support :)

Reply:
Not sure how Spark manages append on WASB.
Hadoop implementation of WASB is designed for single thread append:
https://hadoop.apache.org/docs/current/hadoop-azure/index.html#Append_API_Support_and_Configuration
https://issues.apache.org/jira/browse/HADOOP-12635

Reply:
Interesting, HDFS append implementation guarantees single writer by having a lease on the file, looks like Azure implementation doesn't provide that guarantee..

Reply:
@n3nash fair.. can we create a JIRA for tracking this ? Seems like something we should play with and understand..  I assume you have reading material on the append semantics..

Reply:
https://issues.apache.org/jira/browse/HUDI-341